.go-header.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2022 k0s authors
+Copyright 202. k0s authors
```

Comment:
Oh... it's a regex? Nice. Wasn't clear to me. Have to investigate if that might help fixing https://github.com/k0sproject/k0s/blob/v1.26.1+k0s.0/.golangci.yml#L34-L35 ...

Reply:
That opens up the question if we should escape all the literal dots in the license header :see_no_evil:

Reply:
Good point, I added them.

Reply:
@twz123 I had to remove the escaped dots because it broke the lint. I'm not sure why though.

Reply:
Managed to get it fixed, by adding the regex to .golangci.yml now we only treat year as a regex.